src/Dialect/Mlir/VectorMachineSupport.cpp

```diff
@@ -135,6 +135,7 @@ int64_t Z16VectorMachineSupport::getVectorLength(
     case GenericOps::ConversionGop:
     case GenericOps::CopySignGop:
     case GenericOps::DivGop:
+    case GenericOps::ErfGop:
```

Comment:
@AlexandreEichenberger please check if this place for erf is good or not. Thanks!